save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/pagers/WelcomePager.kt

```diff
@@ -4,14 +4,15 @@ import com.saveourtool.save.frontend.externals.animations.Animation
 import react.ChildrenBuilder
 
 val allWelcomePagers = listOf(
-    listOf(HighLevelSave),
+    // listOf(HighLevelSave),
```

Comment:
Should it be commented?

Reply:
yeah, will think of how to draw it - in the next PR